types/plugin-error/plugin-error-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as PluginError from 'plugin-error';
```

Comment:
Based on the plugin-error documentation, use commonjs module instead:
`import PluginError = require('plugin-error');`

Reply:
done